falcosidekick/README.md

```diff
@@ -44,7 +44,7 @@ Currently available outputs are :
 * [**Apache Kafka**](https://kafka.apache.org/)
 * [**PagerDuty**](https://pagerduty.com/)
 * [**Kubeless**](https://kubeless.io/)
-* [**Kubeless**](https://kubeless.io/)
+* [**OpenFaaS**](https://www.openfaas.com/)
```

Comment:
Can you also add new outputs: RabbitMQ, GCP Storage, AWS S3 (see Readme of falcosidekick)

Reply:
Okay, let me update it.